x-pack/plugins/apm/public/components/alerting/register_apm_alerts.ts

```diff
@@ -68,5 +103,16 @@ export function registerApmAlerts(
       errors: [],
     }),
     requiresAppContext: true,
+    defaultActionMessage: i18n.translate(
+      'xpack.apm.alertTypes.transactionDurationAnomaly.defaultActionMessage',
+      {
+        defaultMessage: `\\{\\{alertName\\}\\} alert is firing for the following configuration:
+
+- Service name: \\{\\{context.serviceName\\}\\}
+- Type: \\{\\{context.transactionType\\}\\}
+- Environment: \\{\\{context.environment\\}\\}
+- Threshold: \\{\\{context.threshold\\}\\}`,
```

Comment:
With anomalies is it still called threshold? And do we have a triggerValue?

Reply:
> With anomalies is it still called threshold? 

I'm not sure what would be a better alternative.

>  And do we have a triggerValue?

Only if we aggregate on the `record_score` field.

Reply:
> I'm not sure what would be a better alternative.

Fair enough. Let's keep `threshold` to stay consistent with the other alerts.

> Only if we aggregate on the record_score field.


Any downsides to doing this? Eg. is it complex or expensive to do? If not we should do it. It's valuable to know the criticality of the alert.

Reply:
@cauemarcondes maybe "severity" makes sense in the case of an anomaly alert.

Reply:
+1 for changing it to severity for the anomaly alerts

Reply:
So `{{severity}}` and `{{triggerValue}}` for anomalies?

Reply:
I think there might be some advantages to keeping the template variables identical between alerts types because it will be possible to re-use the same templates.